plugins/modules/ec2_vpc_dhcp_option_info.py

```diff
@@ -71,35 +71,53 @@
 
 RETURN = '''
 dhcp_options:
-    description: The DHCP options created, associated or found
+    description: The DHCP options created, associated or found.
     returned: always
     type: list
     elements: dict
     contains:
         dhcp_configurations:
-            description: The DHCP configuration for the option set
+            description: The DHCP configuration for the option set.
             type: list
+            elements: dict
+            contains:
+                key:
+                    description: The name of a DHCP option.
+                    returned: always
+                    type: str
+                values:
+                    description: List of values for the DHCP option.
+                    returned: always
+                    type: list
+                    elements: dict
+                    contains:
+                        value:
+                            description: The attribute value. This value is case-sensitive.
+                            returned: always
+                            type: str
             sample:
               - '{"key": "ntp-servers", "values": [{"value": "10.0.0.2" , "value": "10.0.1.2"}]}'
               - '{"key": "netbios-name-servers", "values": [{value": "10.0.0.1"}, {"value": "10.0.1.1" }]}'
         dhcp_options_id:
-            description: The aws resource id of the primary DCHP options set created or found
+            description: The aws resource id of the primary DCHP options set created or found.
```

Comment:
```suggestion
            description: The aws resource id of the primary DHCP options set created or found.
```
This was in the source already, but while we're looking at it and cleaning up docs it would be nice to fix